media/js/firefox/new/scene1.js

```diff
@@ -45,7 +45,8 @@
         if (canReset) {
             $html.addClass('show-refresh');
 
-            $('#refresh-firefox').on('click', function() {
+            $('#refresh-firefox').on('click', function(e) {
```

Comment:
So... just making different IDs is easy enough, but does that mean a bunch of other code duplication as well? Won't I need to repeat all the uitour bits to determine if the client can reset in order to add fire the same event from a second button? 

I'm also not sure how long this will live as a separate script. Once the strings are translated we should be able refactor everything back into a single scene1.js file.

Reply:
Huzzah, this is no longer an issue! The modal doesn't need a refresh button at all, we only need to keep this for the legacy button, which we can also remove once l10n is complete.

Reply:
On further thought we may not even need the legacy button.

Reply:
Old button removed.